openedx/core/djangoapps/course_date_signals/handlers.py

```diff
@@ -63,7 +64,9 @@ def extract_dates_from_course(course):
                     items.extend(next_item.get_children())
     else:
         date_items = []
-        items = modulestore().get_items(course.id)
+        store = modulestore()
+        with store.branch_setting(ModuleStoreEnum.Branch.published_only, course.id):
+            items = modulestore().get_items(course.id)
```

Comment:
Should this line be `store.get_items(course.id)`?

Reply:
Yes, good catch.